src/test/java/testsuite/integration/container/AuroraMysqlIntegrationBaseTest.java

```diff
@@ -0,0 +1,147 @@
+  static final String MYSQL_INSTANCE_4_URL = System.getenv("MYSQL_INSTANCE_4_URL");
+  static final String MYSQL_INSTANCE_5_URL = System.getenv("MYSQL_INSTANCE_5_URL");
+  static final String MYSQL_CLUSTER_URL = System.getenv("DB_CLUSTER_CONN");
+  static final String MYSQL_RO_CLUSTER_URL = System.getenv("DB_RO_CLUSTER_CONN");
+
+  protected static final int MYSQL_PORT = Integer.parseInt(System.getenv("MYSQL_PORT"));
+  protected static final int MYSQL_PROXY_PORT = Integer.parseInt(System.getenv("MYSQL_PROXY_PORT"));
+
+  protected static final String TOXIPROXY_INSTANCE_1_NETWORK_ALIAS = System.getenv("TOXIPROXY_INSTANCE_1_NETWORK_ALIAS");
+  protected static final String TOXIPROXY_INSTANCE_2_NETWORK_ALIAS = System.getenv("TOXIPROXY_INSTANCE_2_NETWORK_ALIAS");
+  protected static final String TOXIPROXY_INSTANCE_3_NETWORK_ALIAS = System.getenv("TOXIPROXY_INSTANCE_3_NETWORK_ALIAS");
+  protected static final String TOXIPROXY_INSTANCE_4_NETWORK_ALIAS = System.getenv("TOXIPROXY_INSTANCE_4_NETWORK_ALIAS");
+  protected static final String TOXIPROXY_INSTANCE_5_NETWORK_ALIAS = System.getenv("TOXIPROXY_INSTANCE_5_NETWORK_ALIAS");
+  protected static final String TOXIPROXY_CLUSTER_NETWORK_ALIAS = System.getenv("TOXIPROXY_CLUSTER_NETWORK_ALIAS");
+  protected static final String TOXIPROXY_RO_CLUSTER_NETWORK_ALIAS = System.getenv("TOXIPROXY_RO_CLUSTER_NETWORK_ALIAS");
+  protected static final int TOXIPROXY_CONTROL_PORT = 8474;
+
+  protected static ToxiproxyClient toxiproxyClientInstance_1;
+  protected static ToxiproxyClient toxiproxyClientInstance_2;
+  protected static ToxiproxyClient toxiproxyClientInstance_3;
+  protected static ToxiproxyClient toxiproxyClientInstance_4;
+  protected static ToxiproxyClient toxiproxyClientInstance_5;
+  protected static ToxiproxyClient toxiproxyCluster;
+  protected static ToxiproxyClient toxiproxyReadOnlyCluster;
+
+  protected static Proxy proxyInstance_1;
+  protected static Proxy proxyInstance_2;
+  protected static Proxy proxyInstance_3;
+  protected static Proxy proxyInstance_4;
+  protected static Proxy proxyInstance_5;
+  protected static Proxy proxyCluster;
+  protected static Proxy proxyReadOnlyCluster;
+  protected static final Map<String, Proxy> proxyMap = new HashMap<>();
+
+  protected final ContainerHelper containerHelper = new ContainerHelper();
+
+  @BeforeAll
+  public static void setUp() throws IOException, SQLException {
+    toxiproxyClientInstance_1 = new ToxiproxyClient(TOXIPROXY_INSTANCE_1_NETWORK_ALIAS, TOXIPROXY_CONTROL_PORT);
+    toxiproxyClientInstance_2 = new ToxiproxyClient(TOXIPROXY_INSTANCE_2_NETWORK_ALIAS, TOXIPROXY_CONTROL_PORT);
+    toxiproxyClientInstance_3 = new ToxiproxyClient(TOXIPROXY_INSTANCE_3_NETWORK_ALIAS, TOXIPROXY_CONTROL_PORT);
+    toxiproxyClientInstance_4 = new ToxiproxyClient(TOXIPROXY_INSTANCE_4_NETWORK_ALIAS, TOXIPROXY_CONTROL_PORT);
+    toxiproxyClientInstance_5 = new ToxiproxyClient(TOXIPROXY_INSTANCE_5_NETWORK_ALIAS, TOXIPROXY_CONTROL_PORT);
```

Comment:
Do we actually need all these? Can we keep them in an array?

Reply:
I was thinking about it and I'm also not a big fun of the way we pass environment variables to a container and how we load them up in a container. This area requires some re-view and re-design. For the moment I'd keep it as is.

Reply:
Sounds good. Created a ticket so we don't forget: https://bitquill.atlassian.net/browse/RDS-625